ext/hook/uhook.c

```diff
@@ -448,6 +448,7 @@ static void dd_uhook_end(zend_ulong invocation, zend_execute_data *execute_data,
             zend_function *orig_func = *(zend_function**)(execute_data->func + 1);
             efree(execute_data->func);
             execute_data->func = orig_func;
+            execute_data->opline = orig_func->op_array.opcodes + orig_func->op_array.last - 1;
```

Comment:
No, this is fine regardless of the version. In fact it's better that we do it, otherwise EX(opline) points to freed memory, in case other observers want to use it.